ledger/participant-integration-api/src/test/lib/scala/platform/store/platform/partymanagement/PersistentPartyRecordStoreTests.scala

```diff
@@ -4,6 +4,7 @@
 package com.daml.platform.store.platform.partymanagement
```

Comment:
This cluster of tests is also in a very strange place:
`com.daml.platform.store.platform.partymanagement` instead of `com.daml.platform.partymanagement`